src/models/GOKU.jl

```diff
@@ -27,6 +40,12 @@ function apply_pattern_extractor(encoder::Encoder{GOKU}, fe_out)
     return pe_z₀_out, pe_θ_out
 end
 
+@doc raw"""
+    apply_latent_in(encoder::Encoder{GOKU}, pe_out)
+Obtains representations of the mean and log-variance of the initial conditions and parameters to use for sampling.
```

Comment:
In order to leave it more general, I would just say "Apply the `encoder`'s `latent_in` layer to `pe_out`, the output of `pattern_extractor`, for obtaining a representation of the mean and squared log-variance of the latent variables."

I would avoid saying initial conditions and parameters because this is specific for GOKU and we will try to provide a general description also valid for LatentODE.